ui/easydiffusion/server.py

```diff
@@ -451,3 +455,26 @@ def modify_package_internal(package_name: str, req: dict):
         log.error(str(e))
         log.error(traceback.format_exc())
         return HTTPException(status_code=500, detail=str(e))
+
+def get_sha256_internal(obj_path):
+    import hashlib
+    from easydiffusion.utils import sha256sum
+
+    path = obj_path.split("/")
```

Comment:
Do we need to check for backslash as well? I've noticed that the model paths with nested subfolders can contain them.

Reply:
Or we could ensure that the JS side never sends backslashes

Reply:
/get/models returns a JSON with nested objects, without any separator. So it's the javascript side that needs to make this into a path.